### ISSUE TITLE
Wondering about this Telegraf error?

### ISSUE THREAD

**Issue by damclarkd** [mihailescu2m/powerwall_monitor#19]
While I have this monitor implemented and running on my RPI, I am seeing errors on the telegraf container logs...

_

> 2021-08-18T14:52:35Z E! [inputs.http] Error in plugin: [url=http://cookieproxy:8675/p/?target=https://powerwall/api/meters/aggregates]: invalid character 'E' looking for beginning of value
> 2021-08-18T14:52:44Z E! [inputs.http] Error in plugin: [url=http://cookieproxy:8675/p/?target=https://powerwall/api/system_status/soe]: Get "http://cookieproxy:8675/p/?target=https://powerwall/api/system_status/soe": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
> 2021-08-18T14:52:44Z E! [inputs.http] Error in plugin: [url=http://cookieproxy:8675/p/?target=https://powerwall/api/meters/aggregates]: Get "http://cookieproxy:8675/p/?target=https://powerwall/api/meters/aggregates": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
> 2021-08-18T14:52:49Z E! [inputs.http] Error in plugin: [url=http://cookieproxy:8675/p/?target=https://powerwall/api/system_status/soe]: Get "http://cookieproxy:8675/p/?target=https://powerwall/api/system_status/soe": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
> 2021-08-18T14:52:49Z E! [inputs.http] Error in plugin: [url=http://cookieproxy:8675/p/?target=https://powerwall/api/meters/aggregates]: Get "http://cookieproxy:8675/p/?target=https://powerwall/api/meters/aggregates": context deadline exceeded (Client.Timeout exceeded while awaiting headers)

_

I'll get several lines of " invalid character 'E' looking for beginning of value" for both aggregates and soe then the timeout errors.
My dashboard is populating with data and I don't see problems other than the errors in the logs. I thought this error was somehow related to JSON errors but just not sure if I need to address or ignore?

Cheers,

**Comment by Zerofire03** [comment]
I've noticed a ton of those errors too. I started noticing them after the most recent update; currently on 21.31.0 Are you still getting data to populate for you in grafana? I looked at the cookiesproxy container, and noticed it is giving a bunch of errors about actually connecting to the Powerwall, so my thoughts so far is that it is an issue with the cookieproxy connecting not allowing it to get the data. I have not been able to track down what is causing it yet, or how to change it, but I'm still looking.

**Comment by damclarkd** [comment]
Yes. Still getting errors, but also still getting all my Grafana data.

**Comment by pridkett** [comment]
I'm looking into this, but I can't quite figure out what is going on. First, as mentioned by other people, here's the errors that I've been seeing in telegraf:

```
2021-08-31T11:18:00Z E! [inputs.http] Error in plugin: [url=http://cookieproxy:8675/p/?target=https://powerwall/api/meters/aggregates]: invalid character 'E' looking for beginning of value
2021-08-31T11:18:00Z E! [inputs.http] Error in plugin: [url=http://cookieproxy:8675/p/?target=https://powerwall/api/system_status/soe]: invalid character 'E' looking for beginning of value
```

I then ran the following commands:

```
raspberrypi01$>   docker exec -it telegraf /bin/bash
root@telegraf:/# curl
curl: try 'curl --help' or 'curl --manual' for more information
root@telegraf:/# curl http://cookieproxy:8675/p/?target=https://powerwall/api/system_status/soe
{"percentage":33.01064393392092}root@telegraf:/#
```

The error comes from golang programs that are trying to unmarshal content.  I've tried it multiple different times and I can't get it to send an improper JSON response that begins with the  `E` character.  I'm not seeing any sort of error on InfluxDB. I'm a little stymied without setting up something to MITM the connection.
